stdlib/REPL/src/REPL.jl

```diff
@@ -208,30 +208,31 @@ end
 
 ==(a::REPLDisplay, b::REPLDisplay) = a.repl === b.repl
 
-function display(d::REPLDisplay, mime::MIME"text/plain", x)
+function display(d::REPLDisplay, mime::MIME"text/plain", @nospecialize(x))
```

Comment:
Does the module level `@nospecialize`:

https://github.com/JuliaLang/julia/blob/44d44708f0f0f2456c623bcd999a89adc35fe8c8/stdlib/REPL/src/REPL.jl#L70

make `@nospecialize` annotations on untyped arguments pointless?